data/pigui/modules/system-view-ui.lua

```diff
@@ -0,0 +1,523 @@
+local Game = require 'Game'
+local Engine = require 'Engine'
+local Event = require 'Event'
+local Lang = require 'Lang'
+local ui = require 'pigui'
+local Format = require 'Format'
+
+local Vector2 = _G.Vector2
+local lc = Lang.GetResource("core")
+
+local player = nil
+local colors = ui.theme.colors
+local icons = ui.theme.icons
+
+local mainButtonSize = Vector2(32,32)  * (ui.screenHeight / 1200)
```

Comment:
Please prefer the use of `ui.rescaleUI` over manually scaling by `ui.screenheight`. Among other reasons, it's easier to grep for the single `ui.rescaleUI` function when we eventually implement resizable windows.

Reply:
I think i fixed it. 704ec4a66cb4f3467d3897b703a73a8687356223